swift/test/Ice/interceptor/Client.swift

```diff
@@ -195,8 +197,14 @@ public class Client: TestHelperI {
             try test(prx.amdAddWithRetry(x: 33, y: 12) == 45)
             try test(interceptor.lastOperation == "amdAddWithRetry")
             try test(interceptor.lastStatus)
+            var ctx: [String:String] = [:]
```

Comment:
`var ctx = ["retry" = "yes"]` should do the job as well

Reply:
It should be `let ctx = ["retry": "yes"]`